src/components/publiq-ui/Page.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const StyledSection = styled.section`
+  background-color: #f0f0f0;
```

Comment:
Just an idea. Not sure if it's the way to go...
Maybe centralize colors in `Theme.js`or `Colors.js` file? 
And use a variable here instead e.g. Colors.lightGrey

What do you think @geoffreydhuyvetters  ?

Reply:
In this case, it might make sense to store them in the theme, yes. Since this is an application-wide thing that will probably not change when we refactor to more primitives.